src/Microsoft.AspNet.OData.Shared/EdmODataAPIHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="EdmODataAPIHandler.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.OData
+{
+
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update.
+    /// This is the handler for data modification where there is no CLR type.
+    /// </summary>
+    public abstract class EdmODataAPIHandler
```

Comment:
Out of curiosity, why is this an abstract class instead of an interface, given that it has no implementation at all?

Reply:
It should be an abstract class similar to `ODataAPIHandler<TStructuralType>`.
What missing is it should implement `IODataAPIHandler` interface.
Fixed